### ISSUE TITLE
where is those Pre-training tasks?

### ISSUE THREAD

**Issue by hscspring** [PaddlePaddle/ERNIE#253]
First of all, many thanks to BaiDu's contribution, you did a really amazing and excellent work.

My question is where to find those pre-training tasks that mentioned in the paper and readme, those are as follows:

- Word-aware Pre-training Tasks
  - Knowledge Masking Task
  - Capitalization Prediction Task
  - Token-Document Relation Prediction Task
- Structure-aware Pre-training Tasks
  - Sentence Reordering Task
  - Sentence Distance Task
- Semantic-aware Pre-training Tasks
  - Discourse Relation Task
  - IR Relevance Task
